doc/source/management/data_at_rest_encryption.rst

```diff
@@ -291,72 +294,58 @@ The implementation of the behavior controlled by this variable is considered
 
-.. rubric:: ON
-
-New tables are created encrypted. You can create an unencrypted table by using
-the ``ENCRYPTION=NO`` clause to the ``CREATE TABLE`` or ``ALTER TABLE``
-statement.
-
-.. rubric:: OFF
-
-By default, newly created tables are not encrypted. Add the ``ENCRYPTION=NO``
-clause in the ``CREATE TABLE`` or ``ALTER TABLE`` statement to create an
-encrypted table.
-
-.. rubric:: FORCE
-
-New tables are created encrypted with the master key. Passing ``ENCRYPTION=NO``
-to ``CREATE TABLE`` or ``ALTER TABLE`` will result in an error and the table
-will not be created or altered.
-
-If you alter a table which was created without encryption, note that it will not
-be encrypted unless you use the ``ENCRYPTION`` clause explicitly.
-
-.. rubric:: KEYRING_ON
-
-:Availability: This value is **Alpha** quality
-
-New tables are created encrypted with the keyring as the default encryption. You
-may specify a numeric key identifier and use a specific ``percona-innodb-`` key
-from the keyring instead of the default key:
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Value
+     - Description
+   * - ON
+     - New tables are created encrypted. You can create an unencrypted table by using
+       the ``ENCRYPTION=NO`` clause to the ``CREATE TABLE`` or ``ALTER TABLE``
+       statement.
+   * - OFF
+     - By default, newly created tables are not encrypted. Add the ``ENCRYPTION=NO``
+       clause in the ``CREATE TABLE`` or ``ALTER TABLE`` statement to create an
+       encrypted table.
+   * - FORCE
+     - New tables are created encrypted with the master key. Passing
+       ``ENCRYPTION=NO`` to ``CREATE TABLE`` or ``ALTER TABLE`` will result in
+       an error and the table will not be created or altered. If you alter a
+       table which was created without encryption, note that it will not be
+       encrypted unless you use the ``ENCRYPTION`` clause explicitly.
+   * - KEYRING_ON (This value is **Alpha** quality)
+     - New tables are created encrypted with the keyring as the default encryption. 
+   * - FORCE_KEYRING (This value is **Alpha** quality)
+     - New tables are created encrypted and keyring encryption is enforced.
+   * - ONLINE_TO_KEYRING (This value is **Alpha** quality)
+     - All tables created or altered without the ``ENCRYPTION=NO`` clause are
+       encrypted with the latest version of the default encryption key. If a table
+       being altered is already encrypted with the master key, the table is recreated
+       encrypted with the latest version of the default encryption key.
+   * - ONLINE_TO_KEYRING_FORCE (This value is **Alpha** quality)
+     - It is only possible to apply the keyring encryption when creating or altering
+       tables.
+
+With the ``KEYRING_ON`` value, **NEW_ID** in ``ENCRYPTION_KEY_ID=NEW_ID`` is an
+unsigned 32-bit integer that refers to the numerical part of the
+``percona_innodb-`` key. You may specify a numeric key identifier and use a
```

Comment:
ENCRYPTION_KEY_ID does not require innodb_encrypt_tables to be specified to any given number - as I described above.